easy-ui-react/src/UnstyledButton/UnstyledButton.tsx

```diff
@@ -52,7 +54,7 @@ export const UnstyledButton = forwardRef<null, UnstyledButtonProps>(
         {...mergeProps(omitReactAriaSpecificProps(restProps), elementProps)}
         disabled={isDisabled}
         ref={mergeRefs(ref, inRef)}
-        className={classNames(className)}
+        className={classNames(styles.UnstyledButton, className)}
```

Comment:
i considered maybe adding it as a prop to control this behavior, but i think it makes a bit more sense as to always have this. if theres a use case im not thinking of though, im open to having a prop

Reply:
having it as a default makes sense to me as well